torchx/schedulers/test/aws_batch_scheduler_test.py

```diff
@@ -342,40 +343,91 @@ def _mock_scheduler_running_job(self) -> AWSBatchScheduler:
+                                        "image": "1234567890.dkr.ecr.us-west-2.amazonaws.com/foo/bar",
+                                        "command": [
+                                            "bash",
+                                            "-c",
+                                            "torchrun ...<omitted for test>...",
+                                        ],
+                                        "environment": [
+                                            {
+                                                "name": ENV_TORCHX_ROLE_NAME,
+                                                "value": "echo",
+                                            }
+                                        ],
+                                    },
+                                }
+                            ],
+                        },
+                        "tags": {
+                            "torchx.pytorch.org/version": "0.3.0dev0",
+                            "torchx.pytorch.org/app-name": "fairseq-train",
+                        },
+                        "platformCapabilities": [],
+                    }
+                ]
+            },
+            {
+                "jobs": [
+                    {
+                        "jobArn": "arn:aws:batch:us-east-1:761163492645:job/7b78f42f-fab7-4746-abb8-be761b858ddb#1",
+                        "jobName": "fairseq-train-wzt5p7p5j3tbqd",
+                        "jobId": "7b78f42f-fab7-4746-abb8-be761b858ddb",
+                        "jobQueue": "arn:aws:batch:us-east-1:761163492645:job-queue/torchx-proto-queue",
+                        "status": "RUNNING",
+                        "attempts": [],  # This is empty on running jobs (unlike completed jobs)
+                        "createdAt": 1656651215531,
+                        "retryStrategy": {
+                            "attempts": 1,
+                            "evaluateOnExit": [{"onExitCode": "0", "action": "exit"}],
+                        },
+                        "startedAt": 1656651662589,
+                        "dependsOn": [],
+                        "jobDefinition": "arn:aws:batch:us-east-1:761163492645:job-definition/fairseq-train-foo:1",
+                        "parameters": {},
+                        "container": {
+                            "logStreamName": "running_log_stream",
+                        },
+                        "nodeProperties": {
+                            "numNodes": 2,
+                            "mainNode": 0,
+                            "nodeRangeProperties": [],
+                        },
+                        "tags": {
+                            "torchx.pytorch.org/version": "0.3.0dev0",
+                            "torchx.pytorch.org/app-name": "fairseq-train",
+                        },
+                        "platformCapabilities": [],
+                    }
+                ]
+            },
+        ]
```

Comment:
can we add test to verify the log streams are actually queried?

Reply:
the test actually does this (but its all mocked so it doesn't actually check logstream names (e.g. the mock CW client just returns whatever it was told to return). I need to make another change to the batch scheduler (for some reason the list API doesn't return all jobs) will revamp the tests to use moto.